docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md

```diff
@@ -40,6 +40,21 @@ If Docker is not installed on the host computer, you need to install a recent ve
 
 If access to the Docker repositories for your Linux distribution is not available on the host computer, you may have to manually transfer the necessary RPM or DEB packages whose locations are specified in [Installing Docker in Airgapped Environments](https://community.replicated.com/t/installing-docker-in-airgapped-environments/81).
 
+If you want to set up Docker on Amazon Linux OS, perform the following:
```

Comment:
IMO, This is too specific a comment to have within the general flow for replicated install. Technically, we asked them to install docker as a pre-req and it is up to them to find the instructions  for their OS rather than us to trying to keep up with docker install methods for each flavor of linux (which is what this will morph into long term)

We could add this instruction to the pre-req page and link to it here. Or maybe to "prepare the environment" https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws/#provision-an-instance-for-yugabytedb-anywhere

Reply:
Thanks @iSignal , Will modify this accordingly.